stellarphot/notebooks/photometry/03-photometry-template.ipynb

```diff
@@ -131,11 +131,11 @@
     "fwhm_estimate = 5\n",
     "\n",
     "# Enter date data was taken here\n",
-    "date = '2021-09-28'\n",
+    "date = ps.ifc.summary['date-obs'][0].split('T')[0]\n",
```

Comment:
Do you want to change the comment above this line, since you are no longer manually setting the date.

Reply:
Excellent.